src/EnergyPlus/Psychrometrics.hh

```diff
@@ -638,17 +648,19 @@ namespace Psychrometrics {
     }
 
 #ifdef EP_psych_errors
-    void PsyWFnTdbH_error(EnergyPlusData &state, Real64 const TDB,             // dry-bulb temperature {C}
+    void PsyWFnTdbH_error(EnergyPlusData &state,
+                          Real64 const TDB,             // dry-bulb temperature {C}
                           Real64 const H,               // enthalpy {J/kg}
                           Real64 const W,               // humidity ratio
                           std::string const &CalledFrom // routine this function was called from (error messages)
     );
 #endif
 
-    inline Real64 PsyWFnTdbH(EnergyPlusData &state, Real64 const TDB,                             // dry-bulb temperature {C}
+    inline Real64 PsyWFnTdbH([[maybe_unused]] EnergyPlusData &state,
+                             Real64 const TDB,                             // dry-bulb temperature {C}
                              Real64 const H,                               // enthalpy {J/kg}
-                             std::string const &CalledFrom = blank_string, // routine this function was called from (error messages)
-                             bool const SuppressWarnings = false           // if calling function is calculating an intermediate state
+                             [[maybe_unused]] std::string const &CalledFrom = blank_string, // routine this function was called from (error messages)
+                             [[maybe_unused]] bool const SuppressWarnings = false           // if calling function is calculating an intermediate state
```

Comment:
I was going to question adding `maybe_unused` to these arguments when they have defaults, but I see now that certain arguments do truly become unused based on the state of the `ifdef`s. That makes sense.